.travis.yml

```diff
@@ -4,42 +4,16 @@ sudo: true
 
 cache: ccache
 
-env:
-  matrix:
-    - CONFIG=DEFAULT
-#    - CONFIG=ALL_ON
-#    - CONFIG=ALL_OFF
+matrix:
+  fast_finish: true
+  include:
 
-addons:
-  apt:
-    packages:
-      - libpcap-dev
-      - libunwind8-dev
-      - lcov
-      - liblua5.2-0
-      - liblua5.2-dev
+    - env: PLATFORM=Unix COMPILER=gcc COVERAGE=1
+      compiler: gcc
+      os: linux
```

Comment:
Are there other platforms supported?

Reply:
The other supported option is osx, which probably isn't necessary at this stage.

I was thinking about doing windows compilations too (they would need to be cross-compiled with mingw). However, I have no idea how to set up cross-compiling toolchains, so I left this out for now

Reply:
Windows has it's own `travis` but i need to fix windows native compilation first (which i started on rewrite).

Reply:
> Windows has it's own travis

which CI does/will it use?

Reply:
It's called AppVeyor https://www.appveyor.com/